### PR TITLE
Fix version inconsistencies in http/*

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Sub-PR of #3368 for the fixes of the files themselves.

### CODE REVIEW

File: http/headers/feature-policy.json [hunk 1]
```diff
@@ -552,7 +552,7 @@
                 "version_added": false
               },
               "webview_android": {
-                "version_added": "57"
+                "version_added": "60"
```
Comment (jpmedley) [inline http/headers/feature-policy.json]: What's your source for this?

**Comment by jpmedley** [comment]
I would need to do some digging. At a gut level, I'd have to say the original values are correct. I submitted the original data while I was working directly with the guys who designed and built feature policy.

**Comment by Elchi3** [comment]
Thanks Joe, I appreciate that as they make no apparent sense right now. Maybe they will with additional explanation.

**Comment by jpmedley** [comment]
It looks like my collaborators weren't looking closely enough at my PRs. Anything earlier than 60 should be set to 60.

**Pull request merged**